### ISSUE TITLE
[FIX] Remove ToSchema from RefundAggregateResponse and exclude it from OpenAPI documentation

### ISSUE THREAD

**Issue by apoorvdixit88** [juspay/hyperswitch#5990]
### Feature Description

The RefundAggregateResponse type used by the [/refunds/aggregate](https://github.com/juspay/hyperswitch/blob/37925626e6446900f1d16e0e5f184ee472d4be3e/crates/router/src/routes/app.rs#L1004) API is currently annotated with ToSchema, which generates OpenAPI schema documentation for it. However, since this API is internal and should not be exposed to external users, the schema should not be generated or included in the OpenAPI documentation.

Current Behavior:
- The `RefundAggregateResponse` struct is annotated with ToSchema, which automatically generates OpenAPI documentation.
- This results in the exposure of internal API details that should remain hidden from external consumers.

Expected Behavior:
- Remove the `ToSchema` derive macro from the RefundAggregateResponse struct.
- Ensure that this struct and its associated API are excluded from OpenAPI documentation generation.

### Possible Implementation

The Api [refunds/aggregate](https://github.com/juspay/hyperswitch/blob/37925626e6446900f1d16e0e5f184ee472d4be3e/crates/router/src/core/refunds.rs#L1098) returns `RefundAggregateResponse`
The following code should be modified to remove ToSchema and update the OpenAPI documentation accordingly:
```
#[derive(Clone, Debug, serde::Serialize, ToSchema)] 
pub struct RefundAggregateResponse {
    /// The list of refund status with their count
    pub status_with_count: HashMap<enums::RefundStatus, i64>,
}
```

Tasks:
- Remove the ToSchema derive macro from the RefundAggregateResponse struct.
- Remove any references to RefundAggregateResponse from openapi.rs to ensure it is not documented.
- Test to verify that this type is no longer included in the OpenAPI schema.

Additional Context:
Since the `/refunds/aggregate` API is for internal use only, removing the OpenAPI schema ensures that this internal API remains undocumented for merchants.

### Have you spent some time checking if this feature request has been raised before?

- [X] I checked and didn't find a similar issue

### Submission Process:
- Ask the maintainers for assignment of the issue, you can request for assignment by commenting on the issue itself.
- Once assigned, submit a pull request (PR).
- Maintainers will review and provide feedback, if any. 
- Maintainers can unassign issues due to inactivity, [read more here](https://github.com/juspay/hyperswitch/wiki/Hacktoberfest-Contribution-Rules).

Refer [here](https://github.com/juspay/hyperswitch/blob/main/docs/TERMS_OF_CONTEST.md) for Terms and conditions for the contest.

**Comment by IshuSinghSE** [comment]
Hello! Could you assign this to me? @apoorvdixit88

**Comment by gorakhnathy7** [comment]
Hey @IshuSinghSE Kindly don't spam the issues with assignment request!

**Comment by IshuSinghSE** [comment]
> Hey @IshuSinghSE Kindly don't spam the issues with assignment request!

Okay

**Comment by gorakhnathy7** [comment]
Thanks for understanding!

**Comment by qstommyshu** [comment]
Hi, I can help on this issue after wrapping up #6117

**Comment by gorakhnathy7** [comment]
Sure @qstommyshu , looking forward to it.

**Comment by RaghavMangla** [comment]
Hi, I would like to work on this issue

**Comment by gorakhnathy7** [comment]
Assigned another issue to you, Thanks for your interest!

**Comment by qstommyshu** [comment]
Hi @gorakhnathy7, 
Would you mind assigning this issue to me? I have a PR #6293 is just waiting to be merged, and I already know how to implement #6111. I just need to wait for #6293 to be merged to start with #6111. Meanwhile I can work on this issue in parallel. 

I want to get at least 4 PR submitted within this month.

**Comment by apoorvdixit88** [comment]
@qstommyshu assigning this issue to you, please open a PR for this, in the mean time your other PRs can get merged.

**Comment by qstommyshu** [comment]
Thanks!
